dsc/src/subcommand.rs

```diff
@@ -272,15 +278,18 @@ pub fn validate_config(config: &str) {
                         },
                     };
                     let properties = resource_block["properties"].clone();
-                    let validation = compiled_schema.validate(&properties);
-                    if let Err(err) = validation {
-                        let mut error = String::new();
-                        for e in err {
-                            error.push_str(&format!("{e} "));
-                        }
-                        error!("Error: Resource {type_name} failed validation: {error}");
-                        exit(EXIT_VALIDATION_FAILED);
-                    }
+                    let _result: Result<(), ValidationError> = match compiled_schema.validate(&properties) {
```

Comment:
Why was this syntax changed from the prior one which is more idiomatic and concise?

Reply:
not sure; maybe rebasing artifact;
Changed it to the original.